django_redis/client/default.py

```diff
@@ -482,6 +482,7 @@ def ttl(self, key, version=None, client=None):
         """
         Executes TTL redis command and return the "time-to-live" of specified key.
         If key is a non volatile key, it returns None.
+        Warning: ttl previously returned an int. Now it returns a float.
```

Comment:
Note: til previously return the number of seconds in an integer. Now it returns the number of seconds with decimals which represent milliseconds in a float.